.github/workflows/bot-pr-updated.yml

```diff
@@ -8,11 +8,11 @@ jobs:
     runs-on: ubuntu-18.04
     if: github.event.pull_request.author_association == 'COLLABORATOR' || github.event.pull_request.author_association == 'MEMBER' || github.event.pull_request.author_association == 'OWNER'
     steps:
-      - name: Checkout the latest code
+      - name: Checkout the latest code (shallow clone)
         uses: actions/checkout@v2
         with:
           ref: develop
-          fetch-depth: 0
+          fetch-depth: 1
```

Comment:
maybe remove it due to the default is 1? same for others

Reply:
good point!